src/wasm-lib/tests/executor/no_visuals.rs

```diff
@@ -95,4 +96,5 @@ gen_test!(if_else);
 //     if_else_no_expr,
 //     "syntax: blocks inside an if/else expression must end in an expression"
 // );
+gen_test_fail!(comparisons_multiple, "syntax: Invalid number: true");
```

Comment:
This message looks like a runtime error. I wouldn't worry too much about it (I don't think we've put much effort into error messages), but it appears like it's executing the first comparison to get `true`.